docs/setting-up-and-running.asciidoc

```diff
@@ -1,115 +1,24 @@
 
 [[setting-up-and-running]]
-== Setting up and running APM Server
+== Setting up APM Server
```

Comment:
I'd consider renaming this section, as generally (and vaguely) "set up" implies install + configure, which is already covered. The best I could come up with is "Operating APM Server", which I don't think is too bad; but maybe you have better ideas (or are happy with "Setting up"!)

Reply:
This section is really tough because it's kind of a mishmash of topics.

**Directory layout** and **Secrets Keystore** could really be at the beginning of the "Configuring APM Server" section. And **APM Server and Systemd** could be in the "Getting Started" section. I understand why **High Availability** was put here initially, but I don't think it really makes sense in this section.

I'm not opposed to the name "Operating APM Server", but I wonder if there's more we can do to clean this up.

Reply:
Much agreed. Feel free to move those sections, in this or other PR